scripts/find_ood_test.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+/*
+ * CSV data format and type definitions
+ */
+
+/* Find_OutOfDate.Go:
+Creates a text file that delineates every criteria which may be out of date based on the commit times to https://github.com/redcanaryco/atomic-red-team/tree/master/atomics
+and https://github.com/secureworks/atomic-validation-criteria/tree/master.
+
+Because we cannot assume that there will be the same number of tests as new tests are added, we will need to mark
+'not found' tests and either prompt the user to generate the test or do it automatically.
+
+In addition, as an unfortunate consequence of how files are stored in the criteria repo https://github.com/secureworks/atomic-validation-criteria,
+we will need to map each test to the file it originated from, since the tests are sometimes stored together (ex. windows/T1027-T1047.csv), and sometimes stored alone (Ex: macos/T1000_macos.csv)
+*/
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+// with the default timeout settings, this test is unable to run to completion. If you would like to test it, either change the default timeout to about 3 minutes,
+// or change the git log command to be since an earlier date
+
+func TestCommitDate(t *testing.T) {
```

Comment:
if a test can't be run repeatedly and quickly, best to remove it or comment it out for future use.  It looks like this test compares commit dates to static date values.

Reply:
Fair point, I wrote that test for the sake of making the program, not really thinking about future changes slowing down compile and run time.

Thanks for the heads up